.gitmodules

```diff
@@ -1,3 +1,21 @@
-[submodule "ext/opencv"]
-	path = ext/opencv
+[submodule "external/zlib"]
+	path = external/zlib
```

Comment:
is it safe to always be on the latest code on the default branch for all of these?  I couldn't find a way to pin to tags/releases - seems a bit risky as a method for dependency management